aspire-update.php

```diff
@@ -70,8 +70,8 @@ function () {
 			'type'      => 'plugin',
 			'slug'      => 'aspireupdate',
 			'version'   => AP_VERSION, // Current version of plugin|theme.
-			'languages' => 'https://github.com/afragen/aspireupdate-translations',
-			'branch'    => 'master',
+			'languages' => 'https://github.com/aspirepress/aspireupdate-translations',
```

Comment:
isn't the repo called translations not aspireupadate-translations?

Reply:
The branch is called `translations` the repo is `aspireupdate-translations` as this might not be the only plugin needing translations.